server/models/HttpError.js

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
A special error class, my thought is: in the model layer, we just need to throw the instance of HttpError, then the controller layer is no need to do anything else then the error would throw to the client, with good response:
The HTTP response code;
And appropriate error message like: the wallet name is incorrect...